Options.lua

```diff
@@ -5771,7 +5780,7 @@ do
 
                                 aoe = {
                                     type = "range",
-                                    name = "AOE Display:  Minimum Targets",
+                                    name = "AOE Display: Show rotation as if there are at least this many targets",
```

Comment:
Still needs something (1) title-cased and (2) concise and (3) stating "priority" (not "rotation") if that verbiage is kept.

"AOE Display: Recommend Using X+ Targets" ?

Below:

> When the AOE display is shown (or AOE mode is active), recommendations will assume there are at least this many targets, even if no additional targets are detected.  If more targets are detected, the actual count will be used.

Might also need explanation as to why you'd want this.  "For ranged specializations using Dual display mode, this setting can help to suggest multi-target enemies before you have damaged them."

Reply:
Title: `Minimum Number of Targets To Count When the Dedicated AOE Display Is Shown`

tooltip
> When the AOE display is shown (or AOE mode is active), its recommendations will assume that there are at least this many targets available. This can be useful with the dual display mode to guarantee your AoE rotation is shown if it doesn't normally change until, for example, 5 targets. Using a setting of 5 would make sure the right priority is followed for "AOE" mode. This can be a different number for different specs and builds.

Reply:
Technically, it functions for both AOE Display and "Primary Display is Forced to AOE Mode".  :D

`Minimum Targets for Dedicated AOE Recommendations` ?

Tooltip is more or less fine.

Reply:
![image](https://github.com/user-attachments/assets/77df09e7-c1d3-463f-baa2-5f170c4cba9d)